src/main/java/com/iexec/blockchain/chain/WebSocketBlockchainListener.java

```diff
@@ -63,7 +63,7 @@ public WebSocketBlockchainListener(final ChainConfig chainConfig, final SignerSe
         final String wsUrl = chainConfig.getNodeAddress().replace("http", "ws");
         this.webSocketService = new WebSocketService(wsUrl, false);
         this.web3Client = Web3j.build(new HttpService(chainConfig.getNodeAddress()),
-                chainConfig.getBlockTime(), Async.defaultExecutorService());
+                chainConfig.getBlockTime().toSeconds(), Async.defaultExecutorService());
```

Comment:
And there was a bug previously here, so
```suggestion
                chainConfig.getBlockTime().toMillis(), Async.defaultExecutorService());
```

Reply:
Resolved in [2104a3a](https://github.com/iExecBlockchainComputing/iexec-blockchain-adapter-api/pull/171/commits/2104a3a6baaa445c32d61938a472a7ae0cdc67c5)